6.3.0 Plugin Release Notes (#769)
128technology/docs: docs/plugin_loopback_static_routes.md

title: Loopback Static Routes Plugin
sidebar_label: Loopback Static Routes
---
:::important

> This plugin has been deprecated starting in SSR 6.3.0.

Reviewer: ```suggestion
This plugin has been deprecated with the release of SSR 6.3.0.
```
